CHANGELOG.md

```diff
@@ -1220,24 +1220,27 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Fix error when submitting an invalid "purpose" step
 - Refactor Commitments:
   - Add `transaction_date` field to be used in place of Financial Year and Quarter
-- Update ISPF themes
```

Comment:
We didn't spot this in the original PR 😅 The latest ISPF themes and non-ODA countries have not been released in release-131.